scripts/pcds_conda

```diff
@@ -128,4 +128,8 @@ source "${TOKEN_DIR}/typhos.sh"
 # The same was done with PyDM and this fixes Designer and friends.
 if [ -n "${SSH_CONNECTION}" ]; then
 	export QT_QUICK_BACKEND="software"
+  # On operator consoles over ssh for most users, opengl crashes unless this
```

Comment:
Duplicating Ken's docs suggestion to the other file
Yes I know it's pretty bad that these files are duplicated like this maybe that's another thing to fix
```suggestion
  # On operator consoles over ssh for most users, opengl crashes unless this is set.
  # This tells the GLX extension to use software rendering by way of the mesa library.
```